frontend/admin/src/js/components/user/ViewUser.tsx

```diff
@@ -32,8 +32,10 @@ export const ViewUserPage: React.FC<ViewUserPageProps> = ({ user }) => {
     description: "Title for the page when viewing an individual user.",
   });
 
-  const userName = `${user?.firstName} ${user?.lastName}`;
-
+  let userName = `${user?.firstName} ${user?.lastName}`;
```

Comment:
It may be worth testing for an empty string, too.  Maybe a helper function in common could help you?

Reply:
Ok. I will do that Thank you!

Reply:
I used isEmpty from "lodash"; is that ok?